src/GroupManager.php

```diff
@@ -93,6 +101,10 @@ public function addGroup($entity_type_id, $bundle_id) {
     $editable->set('groups', $groups);
     $saved = $editable->save();
 
+    // Notify other module we added a new group.
+    // todo: should this be an event?
+    $this->moduleHandler->invokeAll('og_group_created', [$entity_type_id, $bundle_id]);
```

Comment:
Two reasons - allowing other modules act when a group type is created. In D7 you check the field name when the field was created. In here we are creating a new config schema and i'm just abstracting the operation using the hook system.

The second one - this is the most easy way to create the new field without breaking the test.